CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(Slicer_CMAKE_DIR ${Slicer_SOURCE_DIR}/CMake)
 set(Slicer_EXTENSIONS_CMAKE_DIR ${Slicer_SOURCE_DIR}/Extensions/CMake)
 
 #-----------------------------------------------------------------------------
-# http://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/DevelopmentWithGit
+# https://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/DevelopmentWithGit
```

Comment:
Since the current infrastructure redirects link like

```
https://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/DevelopmentWithGit
```

to 

```
https://www.slicer.org/w/index.php/Documentation/Nightly/Developers/DevelopmentWithGit
```

I suggest to update occurrences of `www.slicer.org/slicerWiki` to `www.slicer.org/w`